client/src/store/register/registerFormSlice.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
This is a redux reducer for values that are NOT fetched directly with the api, it allows for easier passing of data between steps, and we are using redux anyway so might as well.